Python/bytecodes.c

```diff
@@ -4074,9 +4078,10 @@ dummy_func(
             assert(oparg == 1);
             STAT_INC(CALL, hit);
             PyObject *res_o = PySequence_Tuple(arg_o);
+            DEAD(null);
```

Comment:
```suggestion
```

No need to declare `null` dead here as it will be handled by `INPUTS_DEAD` below.

Reply:
This is required so that ERROR_IF doesn't emit a useless close for it.